README.md

```diff
@@ -131,14 +137,17 @@ export.status
 
 ### Panels
 
-To get all the panels:
+To paginate all the panels:
 
 ```ruby
-QualtricsAPI.panels.fetch
+panels = QualtricsAPI.panels.fetch
+# => #<QualtricsAPI::PanelCollection:0x007f8769aae2c0 ....>
+
+panels.fetch # get next page
```

Comment:
Ok. Makes sense. And probably we should make it immutable so next/fetch is returning new object. It's also quite odd now... I'll probably divide it to sth like:
* fetch (getting initial result and storing reference to next)
* next (getting next and storing reference to next if exists)